intercom-plugin/plugin.xml

```diff
@@ -65,6 +65,15 @@
               <pod name="Intercom" spec="~> 13.0.0" />
           </pods>
       </podspec>
+
+      <config-file parent="aps-environment" target="*/Entitlements-Debug.plist">
+          <string>development</string>
+      </config-file>
+
+      <config-file parent="aps-environment" target="*/Entitlements-Release.plist">
+          <string>production</string>
+      </config-file>
+
```

Comment:
@MaximBelov thanks for you contribution.
Can you please explain the purpose of this change?
Is it something specific to your own app or is this something you feel should be included as part of the plugin?
Thanks